app/scripts/views/base.js

```diff
@@ -552,7 +552,8 @@ define(function (require, exports, module) {
       err.logged = true;
 
       if (typeof console !== 'undefined' && console) {
-        console.error(err.message || err);
+        var errorMessage = err.errorModule ? err.errorModule.toInterpolatedMessage(err) : (err.message || err);
+        console.error(errorMessage);
```

Comment:
> Is it possible to write a test to check that an error has been logged to the console?

hm can `sinon` spy on `console`?

Sorry what do you mean by `Examples are for checking sentry error logs.`?